pygments/styles/zenburn.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.styles.zenburn
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    Low contrast color scheme Zenburn.
+
+    See: https://kippura.org/zenburnpage/
+         https://github.com/jnurmine/Zenburn
+
+    :copyright: Copyright 2006-2020 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.style import Style
+from pygments.token import (
+    Token, Name, Operator, Keyword, Generic, Comment, Number, String, Literal,
+    Punctuation, Error,
+)
+
+
+class ZenburnStyle(Style):
+    """
+    Low contrast Zenburn style.
+    """
+
+    default_style = ""
+    background_color = '#3f3f3f'
+    styles = {
+        Token: '#dcdccc',
+        Error: '#e37170 bold',
+
+        Keyword: '#efdcbc',
+        Keyword.Type: '#dfdfbf bold',
```

Comment:
Can you please add line number styles here as well? There are four of those in total that should be defined.